docroot/modules/custom/custom_contact/src/Form/ContactSettingsForm.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Drupal\custom_contact\Form;
+
+use Drupal\Core\Form\FormBase;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Configuration form for a contact entity type.
+ */
+class ContactSettingsForm extends FormBase {
```

Comment:
what is purpose of this?

Reply:
It is scaffolded from drush command to generate custom entity.